client/my-sites/marketing/buttons/tray.jsx

```diff
@@ -161,11 +162,21 @@ class SharingButtonsTray extends React.Component {
 	getLimitedButtonsNoticeElement = () => {
 		if ( this.props.limited ) {
 			return (
-				<em className="sharing-buttons-preview__panel-notice">
-					{ this.props.translate( 'Sharing options are limited on private sites.', {
-						context: 'Sharing: Buttons',
-					} ) }
-				</em>
+				<p className="sharing-buttons-preview__panel-notice">
+					<em>
+						{ this.props.translate(
+							'It looks like your site is Private. To get more sharing options make your site Public.',
```

Comment:
I would have probably added a comma to this sentence:

Before: `To get more sharing options make your site Public.`
After: `To get more sharing options, make your site Public.`

Reply:
I took a poll in my previous Happiness team and this is what they agreed on:

```
It looks like your site is Private. Make it Public to take advantage of more sharing options.
```

Thoughts?

Reply:
Yes, sounds great to me!